assets/src/blocks/Accordion/AccordionEditor.js

```diff
@@ -0,0 +1,314 @@
+          accordion_btn_url: attributes.btn_url_5
+        };
+        accordion_rows.push(Object.assign({}, accordion_row));
+
+      }
+
+      return accordion_rows;
+    }
+
+    */
+
+
+  //renders the settings
+  renderEdit() {
+    const { attributes } = this.props;
+
+
+    // let { accordion_rows } = attributes;
+
+    // accordion_rows = this.getAccordionRows(accordion_rows);
+
+    return (
+      <Fragment>
+        <InspectorControls>
+          <PanelBody title={__('Settings', 'planet4-blocks-backend')}>
+            <div>
+              <p> Accordion Block. </p>
+            </div>
+          </PanelBody>
+        </InspectorControls>
+
+        {/*
+        {[accordion_rows].map((item, index) => {
+          return (
+            <div key={index}>
+               { !item.accordion_headline && !item.accordion_text &&
+                <div><b>{__('Please provide a title or a description for this accordion row to be displayed.', 'planet4-blocks-backend')}</b></div>
+              }
+
+            </div>
+          );
+        })}
+      */}
+      </Fragment>
+    );
+  }
+
+
+//renders the editor view
+  renderView() {
+    const { attributes, className } = this.props;
+    console.log(  "Attributes are: " +  this.props.name );
+
+    let { accordion_rows } = attributes;
+    // accordion_rows = this.getAccordionRows(accordion_rows);
+
+    return <Fragment>
+        <section>
+          <div className="block accordion-block my-3">
+            <header>
+              <RichText
+                tagName="h2"
+                className="page-section-header mt-3"
+                placeholder={__('Enter title for the block or leave empty to hide it.', 'planet4-blocks-backend')}
+                value={attributes.accordion_title}
+                onChange={this.toAttribute('accordion_title')}
+                keepPlaceholderOnFocus={true}
+                withoutInteractiveFormatting
+                characterLimit={100}
+                multiline="false"
+              />
+              <RichText
+                tagName="p"
+                className="page-section-description"
+                placeholder={__('Enter the description of the block or leave empty to hide it on the front page.', 'planet4-blocks-backend')}
+                value={attributes.accordion_description}
+                onChange={this.toAttribute('accordion_description')}
+                keepPlaceholderOnFocus={true}
+                withoutInteractiveFormatting
+                characterLimit={400}
+                multiline="false"
+              />
+            </header>
+
+            {[accordion_rows].map((accordion_row, index) => {
+              return (
+                <div key={index}>
+                  <div className="accordion-content my-2">
+                    <div className="accordion card-header"
+                        onClick={this.handleCollapseClick}
+                        id={attributes.accordion_id, index+1}
```

Comment:
Is it wanted here to have two ids? From element inspection it seems only the first one is applied 🤔